paymentsheet/src/main/java/com/stripe/android/paymentsheet/FormFragment.kt

```diff
@@ -4,18 +4,20 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.compose.animation.ExperimentalAnimationApi
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.ComposeView
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.activityViewModels
-import com.stripe.android.paymentsheet.forms.SofortForm
-import com.stripe.android.paymentsheet.forms.SofortFormViewModel
+import com.stripe.android.paymentsheet.forms.Form
+import com.stripe.android.paymentsheet.forms.FormViewModel
 
 class FormFragment : Fragment() {
-    private val sofortFormViewModel by activityViewModels<SofortFormViewModel>()
+    private val viewModel by activityViewModels<FormViewModel>()
```

Comment:
I don't know how it makes the most sense to do this, one fragment per form, and specify the form type here, or specify it in the activity.   I think it will be really obvious when we go to do the integration.